src/Misc/MiddleWare.cpp

```diff
@@ -1535,13 +1537,14 @@ static rtosc::Ports middwareSnoopPortsWithoutNonRtParams = {
         const int slot    = rtosc_argument(msg, 1).i;
 
         int err = 0;
+        printf("save_to_slot: part %d slot %d, filename %s\n", part_id, slot, impl.master->part[part_id]->Pname);
```

Comment:
I'm guessing these printfs were unintentionally left in place.

Reply:
Yes, good catch, this one (line 1540) and the one on line 1543. Do you want to resubmit the PR with those removed (probably best to submit a new one rebased to the latest master)?